src/elements/paginator/common/paginator-common.ts

```diff
@@ -18,14 +18,13 @@ export declare abstract class SbbPaginatorCommonElementMixinType {
   public length: number;
   public pageSize: number;
```

Comment:
shouldn't mixins also declare setter and getter?

Reply:
just checked Lukas's PR, I suppose yes

Reply:
thanks for checking :)